docs/v5/troubleshooting/common-issues.md

```diff
@@ -156,15 +156,83 @@ You may see error messages like the following in your [SchedLog](logs.md#schedlo
 08/30/16 16:53:12 DC_AUTHENTICATE: Command not authorized, done!
 ```
 
+The following are several potential causes and how to check and correct them.
+
+#### Jobs fail to submit: Verify SSL configuration
+
+Your machine must have a valid host certificate and the CE must be
+configured to use it.
+Both the CE and the submitting machines must be configured to recognize
+the Certificate Authorities (CAs) that issued all certificates in use.
+See the documentation about
+[Configuring Certificates](../configuration/authentication.md#configuring-certificates)
+for details.
```

Comment:
I can imagine two different failures here: the server doesn't like the token issuer CA or the client doesn't like the CE's host cert CA. Are there specific errors that we can add here for these cases? What about bad permissions/ownership on the cert and/or key?